lib/add-rm-pkg-deps.js

```diff
@@ -55,7 +55,7 @@ const addSingle = ({pkg, spec, saveBundle, saveType, path}) => {
   pkg[type] = pkg[type] || {}
   if (rawSpec !== '' || pkg[type][name] === undefined) {
     // if we're in global mode, file specs are based on cwd, not arb path
-    pkg[type][name] = specType === 'file' || specType === 'directory'
+    pkg[type][name] = specType === 'file'
```

Comment:
this was introduced in #125 as a way to compensate for the wrong parsed links and should not be necessary anymore